doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -206,6 +206,12 @@ Bluetooth mesh samples
   * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
   * Updated some samples to use DTS overlays instead of KConfig configuration files for setting up external flash memory.
 
+HomeKit samples
+---------------
+* Added:
+
+  * Samples are using Apple HomeKit ADK v6.1
```

Comment:
I don't see any HomeKit samples in the sdk-nrf. This should go into the release notes in the HomeKit repository.

Reply:
Homekit release notes will be now part of nrf Release notes (the same for known issues).

Reply:
Well, but release notes in homekit repository are hard to reach. It is in our documentation in html on the repo - so not very pleasant for the customers to read.

I thought that we agreed that Homekit will be using  known issues and release notes of nrf for publishing major changes